swagger/v1/person_escort_record.yaml

```diff
@@ -0,0 +1,61 @@
+PersonEscortRecord:
+  type: object
+  required:
+    - id
+    - type
+    - attributes
+  properties:
+    id:
+      type: string
+      format: uuid
+      example: f0a91e16-1b0e-4e1f-93fe-319dda9933e6
+      description: The unique identifier (UUID) of this object
+    type:
+      type: string
+      example: person_escort_records
+      enum:
+      - person_escort_records
+      description: The type of this object - always `person_escort_records`
+    attributes:
+      type: object
+      required:
+      - version
+      properties:
+        status:
+          example: in_progress
+          type: string
+          enum:
+            - in_progress
+            - completed
+            - confirmed
+          description: Determines the current status of the `person_escort_record`
```

Comment:
Should this be tagged as `readOnly` attribute?

Reply:
no we will be able to mark something as completed later on

Reply:
Okay, but I'm hoping that will be via a dedicated `POST /per/:id/complete` endpoint ;-)